core/tasty.cabal

```diff
@@ -82,7 +82,7 @@ library
                    wcwidth
     cpp-options: -D__UNIX__
 
-  -- hs-source-dirs:      
+  -- hs-source-dirs:
   default-language:    Haskell2010
   default-extensions:  CPP, ScopedTypeVariables, DeriveDataTypeable
   ghc-options: -Wall
```

Comment:
Can this `Fix` commit be moved earlier?

Reply:
I wanted to keep it close to the `-Werror` addition, for easier upstream consumption. I'll try to sneak in not only our commit style but also our ordering.

Reply:
Fixed.